tests/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+set(
+    sources
+    0001-multiobj.c
```

Comment:
Okay, added:
* https://github.com/edenhill/librdkafka/pull/1014/commits/bb7d7de1a59d96f33fa2c06daf8e8ee54465a2e6

However I have to warn you that it's really not about "I prefer" vs "I not prefer". This is about "plain wrong" vs "right".